### ISSUE TITLE
[Accessibility] Add right-click actions to comment tree

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode-pull-request-github#5698]
Type: <b>Feature Request</b>

CC @meganrogge 

Keyboard users can directly jump to the comment tree via `workbench.action.focusCommentsPanel` command where users can collaps and expand each comment one by one.

To make it more accessible and useful, could you please consider adding the following actions in the context (right-click) menu? FYI, keyboard users can trigger the right-click actions via `Shift+F10` (on Windows; it's cmd+opt+shift+M on Mac for VoiceOver) and this is a widely adopted method among screen reader users.

The actions that I suggest include the following:

- Jump to the range of the current comment applies to

- Delete comment

- Edit comment

- Resolve comment

- Reply to comment

- Reaction to the comment

Extension version: 0.79.2024013010
VS Code version: Code - Insiders 1.86.0-insider (9c5eabb378c9d18a0f712d1f22fe63df3f933ade, 2024-01-30T03:56:13.017Z)
OS version: Windows_NT x64 10.0.22631
Modes:

**Comment by meganrogge** [comment]
I imagine the rationale for this is a user can avoid tabbing through a bunch of elements on the comment widget and instead go through this list with arrow keys.

**Comment by jooyoungseo** [comment]
Yes, that's right! It's quite challenging to locate all the related actions using keyboard. Right-click (AKA, context menu) grouping might be more easily accessible.

**Comment by alexr00** [comment]
We only show the comment thread in the comments view, with a snippet of the first comment in the thread. Because of this, not all the suggested actions make sense:
- Delete comment
- Edit comment
- Reaction to the comment

These ones do make sense on a comment thread, and I will add them:
- Resolve comment (it's actually resolve the comment thread)
- Reply to comment
- Jump to the range of the current comment applies to

**Comment by alexr00** [comment]
> Jump to the range of the current comment applies to

@jooyoungseo, when you hit `enter` or click on a comment thread in the Comments view it already jumps to the comment widget. Does that satisfy what you're looking for here, or do you still think there should be a context menu entry for this?

**Comment by jooyoungseo** [comment]
> @jooyoungseo, when you hit enter or click on a comment thread in the Comments view it already jumps to the comment widget. Does that satisfy what you're looking for here, or do you still think there should be a context menu entry for this?

@alexr00 Yes, it makes sense! Thanks.

**Comment by alexr00** [comment]
Added resolve/unresolve and reply to the context menu. Resolve is also an inline action in the view.
